Marlin/src/pins/stm32f4/pins_FYSETC_S6.h

```diff
@@ -41,17 +41,17 @@
 // EEPROM Emulation
 //
 #if NO_EEPROM_SELECTED
-  #define FLASH_EEPROM_EMULATION
+  //#define FLASH_EEPROM_EMULATION
   //#define SRAM_EEPROM_EMULATION
-  //#define I2C_EEPROM
+  #define I2C_EEPROM
```

Comment:
Perhaps that is all defined in the board's variant file.

Reply:
Probably these frmo the variant.h. I was confused because people in the FR were referring to a different EEPROM type used on the Ender 3 V2.

```
// I2C Definitions
#define PIN_WIRE_SDA            PB9
#define PIN_WIRE_SCL            PB8
```

Reply:
Yes, it works with only I2C_EEPROM.